library/cwm/src/modules/TablePopup.rb

```diff
@@ -799,6 +791,7 @@ def TableHandle(descr, key, event_descr)
           else
             -1
           end
+          # rubocop:enable Lint/DuplicateBranch
```

Comment:
Do you think is it a good opportunity for improving it a bit and stop using those legacies `Ops`?

Reply:
Well, I think TablePopup from CWM is almost not used, so probably not worth it. Also my goal is to not introduce any bugs. So if we improve it, we need also test coverage, which is currently zero.

Reply:
Let's keep it as it is. Thanks.